syncstorage/views/authentication.py

```diff
@@ -20,7 +20,7 @@
 # Coarse validation of FxA userid, device ids, and key ids.
 # This is not supposed to catch all invalid cases, but to act as a backstop
 # that the ids are safe to use and store internally.
-VALID_FXA_ID_REGEX = re.compile("^[A-Za-z0-9=-]{1,64}$")
+VALID_FXA_ID_REGEX = re.compile("^[A-Za-z0-9=-_]{1,64}$")
```

Comment:
I believe the `=-_` sequence there ends up doing a range match for any character between `=` and `-`. This matches some things we don't want (like `@`) and also happens to *not* match the `-` character. Could you please try adjusting this to `=_-` and see if that resolves the CI failures?

Reply:
(You could also escape the `-` to `\-`, which might make it more obvious what's happening here)

Reply:
Sorry for the sloppiness here. Yes, a valid regex should be `=\-_` here, otherwise it's a range. Fixed.